lsstswBuild.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
I don't feel this is necessary for anyone with a basic understanding of `sh`.  "Errorexit is enabled except for when it isn't wanted"?